src/taco-dependency-installer/dependencyInstaller.ts

```diff
@@ -369,15 +374,16 @@ module TacoDependencyInstaller {
                 logger.log(resources.getString("LicenseAgreement"));
             }
 
+            // If we accept prompts automatically, then return immediately
+            if (TacoGlobalConfig.acceptPrompts) {
+                return Q.resolve(true);
```

Comment:
I'm inclined to print out a line stating that the prompt was automatically accepted, rather than skipping it entirely.